source/_integrations/zha.markdown

```diff
@@ -72,6 +72,7 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
   - [Nortek GoControl QuickStick Combo Model HUSBZB-1 (Z-Wave & Zigbee Ember 3581 USB Adapter)](https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/) (Note! Not a must but recommend [upgrade the EmberZNet NCP application firmware](https://github.com/walthowd/husbzb-firmware))
   - [Elelabs Zigbee USB Adapter](https://elelabs.com/products/elelabs_usb_adapter.html)/[POPP ZB-Stick](https://shop.zwave.eu/detail/index/sArticle/2496) (Note! Not a must but recommend [upgrade the EmberZNet NCP application firmware](https://github.com/Elelabs/elelabs-zigbee-ezsp-utility))
   - [Elelabs Zigbee Raspberry Pi Shield](https://elelabs.com/products/elelabs_zigbee_shield.html) (Note! Not a must but recommend [upgrade the EmberZNet NCP application firmware](https://github.com/Elelabs/elelabs-zigbee-ezsp-utility))
+  - Tuya TYGWZ-01 and Lidl Silvercrest Smart Home Gateway (Note! This first have to be [rooted and a serial gateway installed](https://paulbanks.org/projects/lidl-zigbee/ha.html))
```

Comment:
Considering the things that have to be done, I highly doubt we should even advertise this in the Home Assistant documentation.

Reply:
hmm, not quite as streamlined a process yet as with Tasmotizer but otherwise relatively similar to hacking the ITead Sonoff -> https://www.digiblur.com/2020/07/how-to-use-sonoff-zigbee-bridge-with.html a you need USB to TTL to flash MCU on that.

Reply:
My point remains the same.

Reply:
Should I close this until if and when there is a more streamlined process to hack it similar to Sonoff ZBBridge with Tasmotizer?

https://www.digiblur.com/2020/07/how-to-use-sonoff-zigbee-bridge-with.html

Reply:
These are great community guides, we should not send our users into these types or processes.

Reply:
Depending on where you draw the line it could be argued that Sonoff ZBBridge should be removed for the same reasons:

https://www.home-assistant.io/integrations/zha/

ITead Sonoff ZBBridge has two things going against it, firstly it needs to be hacked and secondly, WiFi makes it unstable:

https://github.com/home-assistant/home-assistant.io/issues/17170